lib/geocoder/lookups/location_iq.rb

```diff
@@ -7,23 +7,35 @@ def name
       "LocationIq"
     end
 
+    def use_ssl?
+      true
+    end
+
     def required_api_key_parts
       ["api_key"]
     end
     
     def query_url(query)
-      method = query.reverse_geocode? ? "reverse.php" : "search.php"
-      host = configuration[:host] || "locationiq.org/v1"
```

Comment:
Has the ability to host your own version been removed? Should we still handle that ability since those with local copies would still have them?

Reply:
[Nominatim](http://nominatim.org/) is the one to host and not [LocationIQ](https://locationiq.org/). I see that the host param is implemented https://github.com/alexreisner/geocoder/blob/master/lib/geocoder/lookups/nominatim.rb#L17

However, a drawback with `Nominatim` provider is that it doesn't accept an API key. If `LocationIq` provider is allowed to have `configuration[:host]`, then hosting Nominatim behind an API key is possible. However, some error responses might be different as we have mutated them and added some other info (like our email address.)

Let me know your thoughts on this.

Reply:
I think it would be fine to remove. I will point those who have errors due to this to use the nominatim lookup instead of this one